.github/workflows/ci.yml

```
@@ -0,0 +1,42 @@
+            -destination 'platform=macOS' \
+            CODE_SIGNING_ALLOWED=NO \
+            build | xcpretty
+        env:
+          NSUnbufferedIO: "YES"
+
+      - name: Archive logs on failure
+        if: failure()
+        uses: actions/upload-artifact@v4
+        with:
+          name: xcode-logs
+          path: ~/Library/Logs/DiagnosticReports/*.crash
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)